DOCS/man/input.rst

```diff
@@ -2529,6 +2529,11 @@ Property list
 ``vsync-jitter``
     Estimated deviation factor of the vsync duration.
 
+``display-width``, ``display-height``
+    The current display's horizontal and vertical resolution in pixels. Whether
```

Comment:
Are you 100% sure it's always pixels in all VO backends? Documentation for `osd-dimensions` say
```
usually pixels, but may be scaled pixels with VOs like xv
```
Did you try to confirm that it's still pixels in xv where this scaling applies? Also, windows and OSX also have display scaling, and I don't know if the expected value is the virtual resolution or the actual pixels on screen.

Reply:
the actual. since we don't work with pt anywhere.

Reply:
OK, I also tested on windows, and both `get_working_area` and `get_screen_area` return a size in pixels, apparently regardless of Windows Scaling settings (Windows 10: right-click desktop -> Display Settings -> Advanced scaling settings: tried both custom scale value of 125%, and also tried turning off "Fix scaling for apps" - result remains in pixels).

Reply:
Yeah, it's the actual pixels. The same is true for xv/wlshm and similar VOs.